### ISSUE TITLE
UV expects Rendering property to be in Sequences

### ISSUE THREAD

**Issue by dvhassel** [UniversalViewer/universalviewer#654]
**UV version:**

```
 universalviewer@<version>
```

**I'm submitting a:**

- [X] bug report
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**

According to the Spec (v2.1 and onwards), the Rendering property of a manifest should be in the root.  However, UV expects it to be under Sequences in order to show up in the download-popup

**Expected behavior:**

Extract from the Spec:

 ```
 "rendering": {
    "@id": "http://example.org/iiif/book1.pdf",
    "label": "Download as PDF",
    "format": "application/pdf"
  },
  "within": "http://example.org/collections/books/",

  // List of sequences
  "sequences": [
      {
        "@id": "http://example.org/iiif/book1/sequence/normal",
        "@type": "sc:Sequence",
        "label": "Current Page Order"
        // sequence's page order should be included here, see below...
      }
      // Any additional sequences can be referenced here...
  ]
```

**Steps to reproduce:**

Working manifest in UV:

```
"label" : "Book 1",
"sequences" : [
    {
       "@type" : "sc:Sequence",
	"rendering" : {
		"@id": "http://example.org/iiif/book1.pdf",
		"label" : "Download as PDF",
		"format": "application/PDF"
        },
        "canvases" : [
			{
```

**Comment by irv** [comment]
from the 2.1 spec https://iiif.io/api/presentation/2.1/#rendering
> Any resource type *may* have one or more external `rendering` resources.
from the 3.0 spec https://iiif.io/api/presentation/3.0/#rendering
> Any resource type may have the `rendering` property with at least one item.
> Clients *should* render `rendering` on a `Collection`, `Manifest` or `Canvas`, and *may* render `rendering` on other resource types.

UV3 will render `rendering` on a `range` or a `sequence`. Don't see any reason why we shouldn't support manifest level renderings

**Comment by nicolasfranck** [comment]
for v2 of the presentation api, the UV only uses the first sequence (other sequences are not used).
Given that fact, the manifest rendering can have a total different order of canvases from the order in the first sequence. It can even have canvases that are not in the first sequence (but maybe in the other sequences?). So it doesn't seem like a safe option.

**Comment by edsilv** [comment]
This is fixed in v3.0.36. The UV now checks for sequence, then manifest renderings.
